user-guide/Advanced_Functionality/DataMiner_Agents/Configuring_a_DMA/DMA_configuration_related_to_client_applications.md

```diff
@@ -219,3 +219,22 @@ To use polling (1000 ms) by default, change this as follows:
 ```
 
 For more detailed information on where you can find this file and on the different settings it contains, see [ConnectionSettings.txt](xref:ConnectionSettings_txt#connectionsettingstxt).
+
+## Managing client versions
+
+From DataMiner 10.2.0 \[CU3]/10.2.6 onwards, Cube can automatically update to a more recent version than the DataMiner version installed on the server. This way, users can use the latest Cube features as soon as these are released without having to wait for a server upgrade. However, in the Cube system settings, you can configure limitations on which client version should be used:
+
+1. In DataMiner Cube, go to System Center > *System settings* > *Manage client versions*.
+
+1. Select which client versions should be used:
+
+   - **Allow automatic updates**: This will allow automatic updates to higher versions of Cube, depending on the [configuration in the Cube start window](xref:Opening_DataMiner_Cube#selecting-your-cube-update-track).
+
+   - **Force the matching release version**: This will force users to always use the Cube version that was shipped with the current DataMiner upgrade package.
+
+   - **Force a specific version**: This will force users to always use a particular Cube version, which you will need to select on the server when you choose this option.
+
+> [!NOTE]
+>
+> - To access this configuration, you need the user permissions [Modules > Documents > Add](xref:DataMiner_user_permissions#modules--documents--add), [Modules > System settings > System settings available](xref:DataMiner_user_permissions#modules--system-configuration--system-settings--system-settings-available), and [Modules > System settings > Manage client versions](xref:DataMiner_user_permissions#modules--system-configuration--system-settings--manage-client-versions).
+> - By default, this is configured for the entire DMS. If you select the checkbox *Allow individual agent configuration*, you can configure different client version settings per DMA instead. However, this is not recommended.
```

Comment:
I would further clarify why it is not recommended:
> * ...  However, this is not recommended as users will end up using a different version of Cube depending on which agent they connect to.

An important additional note to make here:
> * When adding a new agent to a cluster, that new agent will come with the default configuration (*Allow automatic updates*) so keep this in mind and apply the same forced version on it after joining the cluster.

This user experience is not ideal and will be further improved upon with DCP184996.